lib/buildTypes/minifyJS.js

```diff
@@ -1,5 +1,4 @@
 var UglifyJS = require("uglify-js");
-var removeSourceMapUrl = require("../remove_source_map_url");
```

Comment:
as i saw, all build process goes through the steam API?
and as i saw, the concatenation is always after transpiling? 

https://github.com/stealjs/steal-tools/blob/2ff7f77322e70cbf7b33484387353ddbcaf088ae/lib/build/multi.js#L89
https://github.com/stealjs/steal-tools/blob/2ff7f77322e70cbf7b33484387353ddbcaf088ae/lib/build/multi.js#L94

do i missed something?

https://github.com/stealjs/steal-tools/blob/792ecdba1b4163518bd3ab3012bc748b4141f350/lib/build/transform.js#L110
https://github.com/stealjs/steal-tools/blob/792ecdba1b4163518bd3ab3012bc748b4141f350/lib/build/transform.js#L140

AND